src/pages/map/components/Picker.jsx

```
@@ -0,0 +1,79 @@
+      const index = Math.round(scrollTop / ITEM_HEIGHT);
+      const targetTop = index * ITEM_HEIGHT;
+
+      ref.current.scrollTo({
+        top: targetTop,
+        behavior: "smooth",
+      });
+
+      const selectedValue = paddedList[index + CENTER_INDEX];
+      if (selectedValue !== "" && index + CENTER_INDEX !== selectedIndex) {
+        setSelectedIndex(index + CENTER_INDEX);
+        onSelectedChange && onSelectedChange(selectedValue);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)